[ContentFinderCondition] Added index 34 for ContentFinderConditionTra…

IcarusTwine opened the pull request:
Added index 34 for ContentFinderConditionTransient also added a default column to link it

IcarusTwine committed:
[ContentFinderCondition] Added index 34 for ContentFinderConditionTransient

Added index 34 for ContentFinderConditionTransient also added a default column to link it

ufx commented on SaintCoinach/ex.json:
```diff
@@ -2756,6 +2756,16 @@
+          "index": 34,
+          "name": "Transient",
+          "converter": {
+            "type": "multiref",
```
This should be a regular "link" type, I don't think it links to anything else

IcarusTwine committed:
Corrected the link instead of a multiref

This however is a reference to diferent things, just not figured out yet
KEEP NOTE
